code/online_challenges/src/project_euler/problem_024/problem_024.py

```diff
@@ -0,0 +1,7 @@
+from itertools import permutations
+def main():
+	result=list(map("".join, permutations('0123456789')))
+	print(result[999999])
```

Comment:
Just move this to one line.

```Python
print(list(map("".join, permutations('0123456789'))))
```
However, I would even recommend that you actually implement the algorithm...since that is the goal of Cosmos and Project Euler. 😉